docs/source/en/optimization/tgate.md

```diff
@@ -0,0 +1,194 @@
+# TGATE
+
+[TGATE](https://github.com/HaozheLiu-ST/T-GATE/tree/main) accelerates inferences of [`PixArtAlphaPipeline`], [`StableDiffusionPipeline`], and [`StableDiffusionXLPipeline`] by skipping the calculation of cross-attention once it converges. More details can be found at [technical report](https://huggingface.co/papers/2404.02747).
+
+![](https://github.com/HaozheLiu-ST/T-GATE/assets/53887227/bff43e0e-2372-4edc-9ba1-64dcbc649329)
+
+
+
+
+## 🚀 Major Features 
+
+* Training-Free.
+* Easily Integrate into [Diffusers](https://github.com/huggingface/diffusers/tree/main).
+* Only a few lines of code are required.
+* Complementary to [DeepCache](https://github.com/horseee/DeepCache).
+* Friendly support [Stable Diffusion pipelines](https://huggingface.co/stabilityai), [PixArt](https://pixart-alpha.github.io/), and [Latent Consistency Models](https://latent-consistency-models.github.io/).
+* 10%-50% speed up for different models. 
+
+## 📖 Quick Start
+
+### 🛠️ Installation
+
+Start by installing [TGATE](https://github.com/HaozheLiu-ST/T-GATE/tree/release-v.0.1.0):
+
+```
+pip install tgate
+```
+
+#### Requirements
+
+* pytorch>=2.0.0
+* diffusers>=0.27.2
+* transformers==4.37.2
+* DeepCache==0.1.1
+* accelerate
+
+### 🌟 Usage
+
+Accelerate `PixArtAlphaPipeline` with TGATE:
```

Comment:
Feel free to update table with any additional `TgateLoader`'s. Also if you can fill in the parentheses with a brief explanation of what the gate step does that'd be great!

```suggestion
To use T-GATE with a pipeline, you need to use its corresponding loader.

| Pipeline | T-GATE Loader |
|---|---|
| PixArt | TgatePixArtLoader |
| Stable Diffusion XL | TgateSDXLLoader |
| Stable Diffusion XL + DeepCache | TgateSDXLDeepCacheLoader |

Next, create a `TgateLoader` with a pipeline, the gate step(`add brief description here`), and the number of inference steps. Then call the `tgate` method on the pipeline with a prompt, gate step, and the number of inference steps.

Let's see how to enable this for several different pipelines.
```